src/MizarData/MizarPairedDataTest.cpp

```diff
@@ -247,7 +253,7 @@ TEST_F(MizarPairedDataTest, ForeachCallstackIsCorrect) {
 TEST_F(MizarPairedDataTest, ActiveInvocationTimesIsCorrect) {
   MizarPairedDataTmpl<MockMizarData> mizar_paired_data(std::move(data_), kAddressToId);
   std::vector<uint64_t> actual_active_invocation_times = mizar_paired_data.ActiveInvocationTimes(
-      {kTID, kAnotherTID}, 1, 0, std::numeric_limits<uint64_t>::max());
+      {TID(kTID), TID(kAnotherTID)}, 1, 0, std::numeric_limits<uint64_t>::max());
```

Comment:
An alternative would be to change the type of the constants to TID and use it with `*kTID` where the raw uint32 value is used. I think this would be more in the spirit of: TID is the correct way of doing things, raw uint32 is the old way.